### PR TITLE
Replaced the deprecated command cve-search to cve

### PR DESCRIPTION
**Pull request by adi88d**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/22941

## Description
- Fixed the script `cveReputation` by using the command `cve` of the integration `CVW Search V2` instead of the deprecated command `cve-search`
- Updated `cve` command to support multiple CVE IDs input.
- CVE Enrichment - Generic playbook -  changed the task `Retrieve CVE reputation` to use cve command with retry-count=5, instead of `cveReputation` script - which do the same.

## Screenshots
image

### CODE REVIEW

File: Packs/CVESearchV2/Integrations/CVESearchV2/CVESearchV2.py [hunk 1]
```diff
@@ -21,11 +21,11 @@ def __init__(self, base_url: str, verify=False, proxy=False):
         super().__init__(base_url=base_url, headers=headers, verify=verify, proxy=proxy)
 
     def cve_latest(self, limit) -> List[Dict[str, Any]]:
-        res = self._http_request(method='GET', url_suffix=f'/last/{limit}')
+        res = self._http_request(method='GET', url_suffix=f'/last/{limit}', timeout=2400)
```
Comment (yaakovi) [inline Packs/CVESearchV2/Integrations/CVESearchV2/CVESearchV2.py:24]: 2400 seconds? why so much?

Comment (adi88d) [reply Packs/CVESearchV2/Integrations/CVESearchV2/CVESearchV2.py]: changed to 60 seconds

**Comment by kirbles19** [comment]
@adi88d Done.

**Comment by ronykoz** [comment]
@yaakovi @adi88d can we merge?

**Comment by adi88d** [comment]
> @yaakovi @adi88d can we merge?

yes, i try to rebuild again

**Pull request merged**